doc/lsst.ap.verify/datasets-install.rst

```diff
@@ -0,0 +1,41 @@
+.. _ap-verify-datasets-install:
+
+###################
+Installing Datasets
+###################
+
+:ref:`ap-verify-datasets` packages data in self-contained units that are intended to be easy to install for LSST Stack users.
+It is not necessary to install all datasets supported by ``ap_verify``, only those you intend to use.
+
+Prerequisites
+-------------
+
+The Dataset framework requires that the computer have version 13.0 or later of the LSST Stack (specifically, the ``obs`` packages and their dependencies) installed.
+`Installing lsst_distrib <https://pipelines.lsst.io/#installation>`_ is the simplest way to ensure all dependencies are satisfied.
+
+The framework also requires `Git LFS`_ and the `EUPS`_ package management system. The latter is usually included by the Stack installation process.
+
+.. _Git LFS: https://developer.lsst.io/tools/git_lfs.html
+.. _EUPS: https://developer.lsst.io/build-ci/eups_tutorial.html
+
+Installation Procedure
+----------------------
+
+Use Git LFS to clone the desired dataset's GitHub repository.
+To get the URL, see the :ref:`package documentation<ap-verify-datasets-index>` for the dataset in question.
+
+.. TODO: should we have a proper versioning system for datasets? (DM-12853)
+
+Once the dataset has been installed, use ``eups declare`` to register the downloaded directory.
+The product name given to EUPS must match the repository name; the version can be anything.
+It is also possible to register the dataset using ``setup``, but this is recommended only for temporary tests.
+
+For example, to install the :ref:`HiTS 2015 <ap_verify_hits2015-package>` dataset,
+
+.. code-block:: sh
+
+   $ git clone https://github.com/lsst/ap_verify_hits2015 mydata
```

Comment:
I've been leaving out any `$` or `>` characters to reduce copy-paste errors. There are special directives for making uncopyable prompts but I've had some trouble with them.